client/components/loading/index.tsx

```diff
@@ -1,15 +1,34 @@
 import { ProgressBar } from '@wordpress/components';
 import clsx from 'clsx';
+import AsyncLoad from 'calypso/components/async-load';
 import './style.scss';
 
 interface LoadingProps {
 	title?: string | React.ReactNode;
 	subtitle?: React.ReactNode;
 	progress?: number;
 	className?: string;
+	useStepContainerV2?: boolean;
 }
 
-const Loading: React.FC< LoadingProps > = ( { title, subtitle, progress, className } ) => {
+const Loading: React.FC< LoadingProps > = ( {
+	title,
+	subtitle,
+	progress,
+	className,
+	useStepContainerV2,
+} ) => {
+	if ( useStepContainerV2 ) {
+		return (
+			<AsyncLoad
+				require="./step-container-v2-loading"
```

Comment:
I think it's bit of a bad idea to async load a loader. What will show when the loader is suspending?